modules/intro/pages/multigraph-overview.adoc

```diff
@@ -1,73 +1,58 @@
+Each graph contains its own data loading jobs and queries, which do not affect and are not visible to other graphs.
 
-[,gsql]
-----
-CREATE GRAPH <gname> (<list of vertex types & edge types>)
-----
 
-[NOTE]
-====
-*MultiGraph Principles*
+== Graph-specific roles and privileges
 
-* A TigerGraph instance with a basic license key can have one graph. A TigerGraph instance with a MultiGraph license key can create multiple graphs.
-+
-**`superuser`**and **`globaldesigner`**can define one or more graphs. The domains of the two graphs may be completely separate, may overlap, or may coincide exactly.
+The TigerGraph system includes several predefined roles.
+Each role is a fixed and logical set of privileges to perform operations.
+In order to access a graph, a user must be granted a role on that graph.
+Without a role, a user has no meaningful access.
 
-* A vertex type or edge type created by a superuser is a global type.
-* A *`superuser`* or *`globaldesigner`* can include a global vertex or edge type in one or more graphs. Global types can be shared among multiple graphs.
-* Users with the *`admin`* or *`designer`* role for a particular graph can add local vertex types and edge types to their own graph. Local types cannot be shared among multiple graphs.
-====
 
-=== Graph-Specific Roles and Privileges
+For details about managing users, privileges, and roles, see xref:user-access:index.adoc[].
 
-The TigerGraph system includes several predefined roles. Each role is a fixed and logical set of privileges to perform operations. In order to access a graph, a user must be granted a role on that graph. Without a role, a user has no meaningful access.
+== Setting a Working Graph
 
-[CAUTION]
-====
-*Role-Based MultiGraph Access Control*
 
-* User roles are granted or revoked on a per-graph basis. Each `GRANT` or `REVOKE` statement specifies not only a role but also a graph.
-* The `GRANT`/`REVOKE` privilege is reserved for superuser and admin users.
-* The *`superuser`* can grant a role to any user on any graph.
-* A *`superuser`* can pick a user to be an *`admin`* on a particular graph. The admin can then manage user privileges on their graph.
-* A user may be granted different roles on different graphs.
-====
+A user must set their working graph in order to access that graph, either using the -g flag with the GSQL command, or by using the `USE GRAPH` command.
+Users who have privileges on more than one graph (including superusers) may only work with one graph at a time. The `GLOBAL SCHEMA_CHANGE JOB` stretches this rule.
 
-For details about managing users, privileges, and roles, see xref:user-access:index.adoc[User Privileges and Authentication]. There you will find a chart describing each of the roles in detail.
 
-=== Setting a Working Graph
+Note that the `CREATE` commands for queries, loading jobs, and schema_change jobs require that the graph name be specified, even for systems with only one graph.
```

Comment:
If you keep the underscore, maybe change to all caps to show that it's command syntax.